spec/lib/navigation/state_file_az_question_navigation_spec.rb

```diff
@@ -58,7 +58,7 @@
     it "returns the correct progress" do
       progress = Navigation::StateFileAzQuestionNavigation.get_progress(StateFile::Questions::FederalInfoController)
       expect(progress).to eq({
-                               title: "Section 5: Complete your state tax return",
+                               title: "Complete your state tax return",
```

Comment:
[microscopic dust] I wonder if it's worth changing "title" to something like "caption" to avoid confusion with the title attribute or page title. Not adamant about this opinion though